tests/ui/push-health/Health_test.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import {
+  render,
+  cleanup,
+  waitForElement,
+  getAllByTestId,
+  queryAllByTestId,
+} from '@testing-library/react';
+
+import Health from '../../../ui/push-health/Health';
+import pushHealth from '../mock/push_health';
+import reposFixture from '../mock/repositories';
+import { getApiUrl } from '../../../ui/helpers/url';
+import { getProjectUrl } from '../../../ui/helpers/location';
+
+const revision = 'cd02b96bdce57d9ae53b632ca4740c871d3ecc32';
+const repo = 'autoland';
+
+describe('Health', () => {
+  beforeAll(() => {
+    fetchMock.get(getApiUrl('/repository/'), reposFixture);
+    fetchMock.get(getApiUrl('/user/'), []);
+    fetchMock.get(getApiUrl('/jobdetail/?job_id=285852375'), {
+      results: [{ id: 285854763, value: 'foo' }],
+    });
+    fetchMock.get(getApiUrl('/jobdetail/?job_id=285854757'), {
```

Comment:
Is the /jobdetail/ endpoint only used to find the `_errorsummary.log` artifacts? If so, they can be retrieved via the taskcluster API, ex: https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/GmleNRBlQSCdSRp8APhGfA/runs/0/artifacts

I'll have a pr for that change shortly (I see it being used for `UnsupportedJob`).

Reply:
Ahh, right.  Yes, that's all we need it for.  I think I can update that here.  We'll need to also download the `task_id` and `run_id`.

Reply:
Actually, I'm going to leave the `jobdetail' part as-is for now.  But the I updated so the `job` objects we download will have a `task_id` and `run_id` in them for when you update your PR.  I figure you have a constant somewhere for accessing that URL.  I don't want to repeat that or get it wrong.  :)

Reply:
Excellent, that'll work.